build.sh

```diff
@@ -52,11 +51,8 @@ function build_otel() {
 function build_otel_jmx() {
   cd submodules/opentelemetry-java-contrib
   mkdir -p "$DESTDIR$subagentdir/opentelemetry-collector/"
-  # Build & test systems do not always check out git history for submodules, so the properties assigned
-  # here allow the nebula release process to function properly in that state
-  ./gradlew --no-daemon -Pgit.root="$(pwd)/../../.git/" -Prelease.version=${JMX_METRICS_JAR_VERSION} -Prelease.disableGitChecks=true :jmx-metrics:build
-  # TODO: Parameterize this jar name once we can control the release artifact
-  cp "jmx-metrics/build/libs/opentelemetry-jmx-metrics-${JMX_METRICS_JAR_VERSION}.jar" "$DESTDIR$subagentdir/opentelemetry-collector/opentelemetry-java-contrib-jmx-metrics.jar"
+  ./gradlew --no-daemon :jmx-metrics:build
```

Comment:
Maybe I'm missing some context, but why were these changes needed here?

Reply:
java-contrib maintainers removed the "nebula" gradle release plugin that interacted with those flags.

Reply:
Removed here: https://github.com/open-telemetry/opentelemetry-java-contrib/pull/274

Reply:
This broke the Windows build, because `Dockerfile.windows` hasn't been updated.